imops/morphology.py

```diff
@@ -517,3 +522,82 @@ def distance_transform_edt(
         return result[0]
 
     return None
+
+
+def convex_hull_image_2d(image, offset_coordinates=True):
```

Comment:
just call it `convex_hull_image` and fail at runtime if the image is not 2D

Reply:
Done